ConsoleApp8/Program.cs

```diff
@@ -1,12 +1,81 @@
-﻿using System;
+﻿using ConsoleApp8.Models;
+using System;
+using System.IO;
 
 namespace ConsoleApp8
 {
-    class Program
+    public class Program
     {
+        public const int MAX_ATTEMPTS = 10;
+        const string InitialDataFile = "customers.json";
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string jsonText = File.ReadAllText(InitialDataFile);
+            Console.WriteLine("JSON string is \n{0}", jsonText);
+            CustomersModel jsonObject = jsonText.As<CustomersModel>();
+
+            PrintShipmentOrders(SetIDForCustomerShipmentOrder(jsonObject));
+
+            Console.WriteLine("The End");
+            Console.ReadLine();
+        }
+
+        public static CustomersModel SetIDForCustomerShipmentOrder(CustomersModel jsonObject)
+        {
+            for (int customerElement = 0; customerElement < jsonObject.Customers.Count; customerElement++)
+            {
+                jsonObject.Customers[customerElement].ID = Guid.NewGuid();
+                for (int shipmentElement = 0; shipmentElement < jsonObject.Customers[customerElement].Shipments.Count; shipmentElement++)
+                {
+                    jsonObject.Customers[customerElement].Shipments[shipmentElement].ID = (shipmentElement + 1).ToString();
+
+                    for (int orderElement = 0; orderElement < jsonObject.Customers[customerElement].Shipments[shipmentElement].Orders.Count; orderElement++)
+                    {
+                        jsonObject.Customers[customerElement].Shipments[shipmentElement].Orders[orderElement].ID = orderElement + 1;
+                    }
+                }
+            }
+            return jsonObject;
+        }
+
+        public static void PrintShipmentOrders(CustomersModel jsonObject)
+        {
+            Validator validInput = new Validator();
+            int attempt = 0;
+            while (attempt < MAX_ATTEMPTS)
+            {
+                switch (validInput.GetChoiceToShowShipmentInfo())
+                {
+                    case 1:
+                        for (int customerElement = 0; customerElement < jsonObject.Customers.Count; customerElement++)
```

Comment:
please, move the actions under the "case 1" and "case 2" to a separate method, add a parameter that defines if the details should be printed or not.
Please, define "ToString()" methods in the models classes and use them in this method

Reply:
try to use linq instead of for loop here